rust/stackablectl/Cargo.toml

```diff
@@ -29,5 +29,9 @@ serde.workspace = true
 snafu.workspace = true
 tera.workspace = true
 tokio.workspace = true
+tokio.features = ["io-std"]
```

Comment:
suggestion: Does it maybe make sense to use the object notation instead? It would be then inline with the root `Cargo.toml` file.

Reply:
Or we can move it into the root Cargo.toml.

Reply:
Sure, that's also possible!

Reply:
3fa1b7ee84a851a54c4b2bb4a5281ee52606cf0a